packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -95,6 +95,7 @@ contract OptimisticGovernor is Module, Lockable {
             bytes32 _identifier,
             uint64 _liveness
```

Comment:
OZ commented on the 0-bond case. If we don't intend to block a 0 bond, we should probably at least add a comment explaining that a 0 bond is allowed, but will mean that there is no reward for disputes.